.github/actions/chatbot/action.yaml

```diff
@@ -0,0 +1,52 @@
+name: chatbot
+description: Recreate Chatbot LLamaindex Vector Index
+
+inputs:
+  iam_role:
+    description: 'The IAM Role to use to perform the deploy'
+    required: true
```

Comment:
This whole part of the pipeline will be run on a self hosted runner on AWS CodeBuild.
CodeBuild has its own role that can be used to contact AWS services
For this reason, it is not necessary to set IAM credentials/roles

```suggestion
```